media/js/firefox/welcome/welcome10-experiment.js

```diff
@@ -0,0 +1,37 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+(function (Mozilla) {
+    'use strict';
+    /* update dataLayer with experiment info */
+    var href = window.location.href;
+
+    var initTrafficCop = function () {
+        if (href.indexOf('v=') !== -1) {
+            if (href.indexOf('v=a') !== -1) {
+                window.dataLayer.push({
+                    'data-ex-variant': 'experiment-variant-1',
+                    'data-ex-name': 'experiment-headline-variant'
```

Comment:
Should the experiment name include `welcome-10` so we know what page it relates to?